corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -39,28 +39,28 @@ hqDefine("scheduling/js/conditional_alert_main", [
             var newRuleName = $("#id_conditional-alert-name").val();
             if (!name) {
                 $("#conditional-alert-form").submit();
-            } else if (name !== newRuleName) {
-                var caseType = $(
-                    "#select2-id_criteria-case_type-container"
-                ).text();
-                var caseCountUrl = $("#conditional-alert-case-count-url").attr(
-                    "url"
-                );
+
+            } else {
+                var caseType = $("#id_criteria-case_type").val();
+                var caseCountUrl = initialPageData.reverse('count_cases_by_case_type');
+                $("#conditional-alert-save-btn").html('<i class="fa fa-spin fa-spinner"></i>');
```

Comment:
This adds a spinner but doesn't prevent the user from re-clicking. Can it use `disableButton` instead, which will handle both?

Reply:
It'll also save you from needing to reset the button text below.